scout/server/templates/utils.html

```diff
@@ -42,7 +42,7 @@ <h6 class="comments-title ml-3 mt-2"></i><i class="far fa-comments"></i> Comment
     <style>.odd { background-color: #d5e8f0 }</style>
     {% for comment in comments %}
       <div class="row ml-1 alert {% if loop.index0 % 2 %}even{% else %}odd{% endif %}">
-          <div class="col-7 text-break">{{ comment.content|fix_punctuation }}</div>
```

Comment:
That function was introduced to prevent [a bug](https://github.com/Clinical-Genomics/scout/issues/2451). It inserts a space after .,:;?!
Perhaps we could remove the space after the dot instead?

Reply:
Bah perhaps the idea was that with spaces the comments would be more readable instead of breaking every 50 or whatever chars